### ISSUE TITLE
[Models][UX]: Design Icon for ArgoCD Component - ArgoCD Export

### ISSUE THREAD

**Issue by ritiksaxena124** [meshery/meshery#10294]
### Description
Meshery is a powerful platform that contains over [200 integrations](https://layer5.io/cloud-native-management/meshery/integrations) for managing cloud-native applications. MeshMap is an extension of Meshery that visually represents these integrations and components. Currently, most of the icons used in MeshMap have the default shape of a circle. However, to improve visual clarity and consistency, we need to design custom icons for MeshMap components.

### Desired Behavior
A new Icon for  **ArgoCD Export** needs to be created. This issue calls for a volunteer to draft a new Icon.

### Instructions for Designing a Component Icon
1. Create two versions of the icon one is white and another one is colored
2. Provide the icon in SVG format
3. Make a PR after adding the icon inside the [new-icon folder](https://github.com/meshery/meshery/tree/master/ui/assets/new-icons) present in the meshery repository
4. Make sure the icon for the component is different from the Model icon

### How to check whether the icon you designed is the same as the Model icon or any other component icon?
1. Go to the [playground](https://playground.meshery.io/extension/meshmap) -> search for the component you are trying to design an icon for and check whether the icon that you are designing is already present for another component or not if not then you can make a PR.
7. If you don't have access to Playground then you have to fill out a [signup](https://play.meshery.io/) and get verified before you access the Playground by the time you get access, you can go to [integrations](https://layer5.io/cloud-native-management/meshery/integrations) page in layer5.io site and search for the component and see whether an icon is already present or not.
8. 
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- [MeshModel Repository](https://github.com/meshery/meshery/tree/master/server/meshmodel)
- 🖥 [MeshModel Overview](https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.g226f5de5180_19_259)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by octonawish-akcodes** [comment]
can i work on this? @ritiksaxena124

**Comment by ritiksaxena124** [comment]
Go ahead @octonawish-akcodes

**Comment by Aviral0702** [comment]
I am starting to work upon this issue. Please assign it to me.

**Comment by Aviral0702** [comment]
@vishalvivekm Please assign the issue to me.

**Comment by JAYANTJOSHI001** [comment]
@ritiksaxena124 @leecalcote pls assign this issue to me.

**Comment by KKSurendran06** [comment]
It seems like this issue has been stalled for some time. I would like to work on it and will make a PR in the coming days

**Comment by srujanaelicherla** [comment]
Hi @leecalcote @ritiksaxena124,

I’m interested in working on this issue! My approach would be:
Design two versions of the ArgoCD Export icon (white and colored) in SVG format.
Ensure the icon is visually distinct from other component and model icons by checking the Playground and integrations page.
Add the icon to the new-icon folder in the repository and create a PR once ready.

I’d love to be assigned this issue and contribute to improving the visual clarity and consistency of MeshMap components.

Thank you!

**Comment by srujanaelicherla** [comment]
Hi @leecalcote,

PR #15770 is ready for review, all CI checks are now passing. Please take a look at it when you get a minute.

Thank you!

**Comment by Sekar-C-Mca** [comment]
@leecalcote @ritiksaxena124 
If this issue is still open, could you please assign it to me?

**Comment by leecalcote** [comment]
@Sekar-C-Mca, yes, find another model that still has the same icon for every component. Each component should have different icons.

**Comment by Sekar-C-Mca** [comment]
@leecalcote  KubeGems,KubeDB,Istio,Fluent,Loki has the same icon for every component.

**Comment by jamieplu** [comment]
@Sekar-C-Mca, no kidding. Get busy.

**Comment by suzy-g38** [comment]
@saurabhraghuvanshii if this issue is relevant, this can be a good-first-issue and you can remove the assignee

**Comment by aryanghai12** [comment]
can i work on this @saurabhraghuvanshii?
